customize.html

```diff
@@ -69,84 +63,81 @@ <h3>Map generation</h3>
       <li><a href="https://forum.minetest.net/viewtopic.php?t=4394">More Trees</a></li>
       <li><a href="https://forum.minetest.net/viewtopic.php?t=5790">Nether</a></li>
       <li><a href="https://forum.minetest.net/viewtopic.php?t=3898">Plantlife</a></li>
-      <li><a href="https://forum.minetest.net/viewtopic.php?t=2290">Snow Biomes</a></li>
+      <li><a href="https://forum.minetest.net/viewtopic.php?t=18413">Cave Realms Lite</a></li>
     </ul>
   </div>
-</div>
-
-<div class="row">
-  <div class="col-sm-6">
-    <h3>Environment</h3>
+  <div class="col-sm-4">
+    <h3>Player effects</h3>
     <ul>
-      <li><a href="https://forum.minetest.net/viewtopic.php?t=2807">Immersive Sounds</a></li>
-      <li><a href="https://forum.minetest.net/viewtopic.php?t=5245">Weather</a></li>
+      <li><a href="https://forum.minetest.net/viewtopic.php?t=4654">3D Armor</a></li>
+      <li><a href="https://forum.minetest.net/viewtopic.php?t=4870">Achievements</a></li>
+      <li><a href="https://forum.minetest.net/viewtopic.php?t=16913">Minecraft-like item drop/pick-up</a></li>
+      <li><a href="https://forum.minetest.net/viewtopic.php?t=16365">Throwing Redo</a></li>
+      <li><a href="https://forum.minetest.net/viewtopic.php?t=9650">Sprint</a></li>
+      <li><a href="https://forum.minetest.net/viewtopic.php?t=11153">HUD Bars</a></li>
+      <li><a href="https://forum.minetest.net/viewtopic.php?t=11336">HBHunger</a></li>
     </ul>
   </div>
+</div>
 
-  <div class="col-sm-6">
-    <h3>Plants &amp; Mobs</h3>
+<div class="row">
+  <div class="col-sm-4">
+    <h3>Plants, Animals &amp; Monsters</h3>
     <ul>
       <li><a href="https://forum.minetest.net/viewtopic.php?t=9019">Farming Redo</a></li>
       <li><a href="https://forum.minetest.net/viewtopic.php?t=629">Animals Modpack</a></li>
-      <li><a href="https://forum.minetest.net/viewtopic.php?t=8638">Creatures Mob Engine</a></li>
```

Comment:
perhaps suggest [this](https://github.com/minetest-mods/mob-engine) instead?
Though it doesn't seem to have a forum topic

Reply:
For consistency, I've disregarded multiple forks which do not have forum topics. I think it's best just for the author to make a topic for them